doc/sphinx/source/frameworks/generic.rst

```diff
@@ -89,7 +89,10 @@ Memory allocation
 -----------------
 
 Buffer pointers should be aligned at the 64-byte boundary. NUMA policy should be 
```

Comment:
Instead of `at the 64-byte boundary`, `on 64-byte boundaries`

Reply:
fixed